data/plugins/editor/README.md

```diff
@@ -0,0 +1,167 @@
+    esac
+  fi
+}
+
+toggle_window() {
+  local tag cmd target current_window
+  tag="$1"
+  cmd="$2"
+  target="$(tmux show -qv "@$tag")"
+  current_window="$(tmux display -p '#S:#I')"
+
+  if [ "$current_window" = "$target" ]; then
+    tmux last-window
+  else
+    if [ -n "$target" ] && tmux lsw -F'#S:#I' | grep -q "$target"; then
+      tmux select-window -t "$target"
+    else
+      if [ -z "$cmd" ]; then
+        target="$(tmux new-window -PF'#S:#I')"
+      else
+        target="$(tmux new-window -PF'#S:#I' -- sh -c "$cmd")"
+      fi
+      tmux set "@$tag" "$target"
+    fi
+  fi
+}
+
+toggle_pane() {
+  local tag cmd size target current_window
+  tag="$1"
+  cmd="$2"
+  size="$3"
+  target="$(tmux show -qv "@$tag")"
+  current_window="$(tmux display -p '#S:#I')"
+
+  if [ "$current_window" = "${target%.*}" ]; then
+    target="$(tmux break-pane -dPs "$target")"
+  else
+    if [ -n "$target" ]; then
+      tmux join-pane -bhl "$size" -s "$target" -t "$current_window.0"
+      target="$current_window.0"
+    else
+      if [ -z "$cmd" ]; then
+        target="$(tmux split -bhl "$size" -PF'#S:#I.#P')"
+      else
+        target="$(tmux split -bhl "$size" -PF'#S:#I.#P' -- sh -c "$cmd")"
+      fi
+      tmux select-pane -t "$target" -P 'bg=#090B10'
+    fi
+  fi
+  tmux set "@$tag" "$target"
+}
+
+main "$@"
+```
+</details>
+
+
+<details>
+  <summary>tmux-toggle-pane-vifm</summary>
+
+```bash
+#!/bin/bash
+
+tmux-toggle pane fm 'vifm -c "set vicmd='\''#editor#run helix_tmux'\''" "$(pwd)"' 25%
+```
+</details>
+
+#### 3. Configure Tmux
+
+<details>
+  <summary>tmux.conf</summary>
+
+```shell
+bind -n 'c-b' run-shell 'tmux-toggle-pane-vifm'
+bind -n 'c-t' run-shell 'tmux-toggle window term'
+
+# Theming
+set -g default-terminal "tmux-256color"
+set -as terminal-overrides ",alacritty*:Tc"
+set -g status off
+set -g pane-border-status off
+set -g pane-border-indicators off
+set -g pane-active-border-style "fg=#0F111A"
+set -g pane-border-lines heavy
+```
+</details>
+
+#### 4. Configure Vifm:
+
+<details>
+  <summary>vifmrc</summary>
+
+```vimscript
+" Hides information when there is not enough space. Customize it however you want.
+if &columns > 160
+    vsplit | view!
+    set viewcolumns=-{name}..,7{size},12{perms},7{uname}..,-7{gname}..
+    set statusline="%1* %t %2* %3*%{filetype('.')} %T %= %a / %c | %d %2*%1* %E "
+elseif &columns > 81
+    only
+    set viewcolumns=-{name}..,7{size},12{perms},7{uname}..,-7{gname}..
+    set statusline="%1* %t %2* %3*%{filetype('.')} %T %N%= %a / %c | %d %2*%1* %E "
+else
+    only
+    set viewcolumns=-{name}..,
+    set statusline=" "
+    set rulerformat=" "
+endif
+
+" Toggles the preview window
+command toggle :
+\|  if layoutis('only')
+\|    restart
+\|  else
+\|    view
+\|    only
+\|  endif
+
+" Good to have keybindings
+noremap T :tree depth=2<cr>
+noremap B :toggle<cr>
+noremap E :!nohup $TERMINAL -e tmux-ide %f &>/dev/null & disown<cr>
+```
+</details>
```

Comment:
I think you forgot to say how to use it: that `tmux-ide` is the main script to be run from the shell or via that <kbd>E</kbd> mapping.

Reply:
I added a description.

Reply:
Yes, I've just added the description.